viewing_party/main.py

```diff
@@ -0,0 +1,131 @@
+# WAVE One
+def create_movie(movie_title, genre, rating):
+    movie = {}
+    movie['title'] = movie_title
+    movie['genre'] = genre
+    movie['rating'] = rating
+
+    for values in movie.values():
+        if values == None:
+            return None
+
+    return movie
+
+
+def add_to_watched(user_data, movie):
+    new_entry = user_data["watched"]
+    new_entry.append(movie)
+    return user_data
+
+
+def add_to_watchlist(user_data, movie):
+    new_entry = user_data["watchlist"]
+    new_entry.append(movie)
+    return user_data
+
+
+def watch_movie(user_data, title):
+    for movie in user_data['watchlist']:
+        if movie['title'] == title:
+            user_data['watched'].append(movie)
+            user_data['watchlist'].remove(movie)
+            return user_data
+    else:
+        return user_data
+
+
+# Wave Two
+
+def get_watched_avg_rating(user_data):
+    if len(user_data["watched"]) == 0:
+        return 0.0
+    else:
+        total = 0
+        for movie in user_data["watched"]:
+            total += movie["rating"]
+        average_rating = total / len(user_data["watched"])
+    return average_rating
```

Comment:
this works perfectly! but let's see if we can use only one `return` statement:

```suggestion
    total = 0.0
    if len(user_data["watched"]) > 0:
        for movie in user_data["watched"]:
            total += movie["rating"]
        total = total / len(user_data["watched"])
    return total
```